server/src/main/java/com/decathlon/ara/service/ProjectService.java

```diff
@@ -164,11 +164,11 @@ private void validateUniqueName(ProjectDTO dto) throws NotUniqueException {
         }
     }
 
-    private void validateOnlyOneDefault(ProjectDTO dto) throws NotUniqueException {
+    private void validateOnlyOneDefault(ProjectDTO dto)  {
```

Comment:
If you change this method to pass it from "raise an exception because rules is invalid" to "silencely change the default project", I think that this method can be rename since it doesn't validate anymore.